text/0635-mediaconvert-l2-construct.md

```diff
@@ -0,0 +1,258 @@
+simplifies the setup process, allowing you to focus on other aspects of your media processing pipeline.
+
+#### Presets
+
+To create a Custom Preset using an L2 Construct, you define the ``presetProps`` object, which includes all the necessary 
+settings for your media processing tasks. This object is then passed to the ``Preset`` construct to create the custom preset.
+
+    const presetProps: PresetProps = {
+        name: 'CustomPreset',
+        settings: {
+            VideoDescription: {
+                CodecSettings: {
+                    Codec: 'H_264',
+                    H264Settings: {
+                        MaxBitrate: 5000,
+                        RateControlMode: 'CBR',
+                        FramerateControl: 'SPECIFIED',
+                        FramerateNumerator: 30,
+                        FramerateDenominator: 1
+                    }
+                }
+            },
+            AudioDescriptions: [
+                {
+                    CodecSettings: {
+                        Codec: 'AAC',
+                        AacSettings: {
+                            Bitrate: 128000,
+                            CodingMode: 'CODING_MODE_2_0',
+                            SampleRate: 44100
+                        }
+                    }
+                }
+            ]
+        }
+    };
+
+    // Define the custom MediaConvert Preset
+    const customPreset = new Preset(this, 'CustomPreset', presetProps);
+
+In this example, the ``customPreset`` is configured with specific video and audio encoding settings. By defining these properties, you can create a Preset that matches your exact media processing requirements, allowing for consistent and optimized encoding across your jobs.
+
+System Presets are a collection of pre-configured and managed Presets provided by AWS Elemental MediaConvert. These presets are designed to handle common media processing scenarios and come with recommended best practices for video and audio encoding. By using System Presets, you can quickly set up your MediaConvert jobs by simply referencing the preset's name, ensuring that your encoding tasks are performed with optimized settings.
+
+In this example, the GenericMp4Hev1AvcAacPreset is a predefined System Preset. By specifying its name, you automatically use a set of optimal encoding settings tailored for a specific media conversion scenario. This simplifies the configuration process, allowing you to focus on other critical aspects of your media processing pipeline.
+
+    const genericMp4Hev1AvcAacPreset = new Preset(this, 'GenericMp4Hev1AvcAacPreset', {
+        name: "GenericMp4Hev1AvcAacPreset"
+    });
+
+Using System Presets like this allows you to leverage AWS MediaConvert’s best practices without needing to manually configure every encoding parameter, making your media processing workflows more efficient and reliable.
+
+
+
```

Comment:
Please add RFC for L2 Queues

Reply:
Sure, I will add them in my next commit.